webrtc-stats.html

```diff
@@ -622,8 +622,43 @@ <h3>
           the first RTCP packet is sent or received that refers to the <a>SSRC</a> of the <a>RTP stream</a>.
         </p>
         <p>
-          RTP monitored objects are not [= deleted =].
+          RTP monitored objects are deleted when the corresponding RTP sender or
+          RTP receiver is reconfigured to remove the corresponding RTP stream.
+          This happens for the old SSRC when the {{RTCRtpStreamStats/ssrc}}
+          changes, a simulcast layer is dropped or the {{RTCRtpTransceiver}}'s
+          {{RTCRtpTransceiver/currentDirection}} becomes <code>"stopped"</code>.
+          The monitored object is <i>not</i> deleted if the transceiver is made
+          <code>"inactive"</code> or if the encoding parameters'
+          {{RTCRtpEncodingParameters/active}} attribute is set to
+          <code>false</code>. If an SSRC is recycled after a deletion event has
```

Comment:
Good to avoid "event" I think
```suggestion
          <code>false</code>. If an SSRC is recycled after deletion has
```